### ISSUE TITLE
Pocket option

### ISSUE THREAD

**Issue by andreker00** [mdn522/binaryapi#22]
Hello, can you get the pocket option broker api?  It is also a large broker and allows copy trade from other users

**Comment by mdn522** [comment]
PocketOption does not provide any public API. to make one. we have to decode/reverse engineer their website and replicate it in Python. I did put some of my time in 2020 into creating PocketOption API. but it was not enough (time). The login system (for WebSocket) but might be tricky. but the rest of them should not be that tough. You can start by going to the browser's dev tools network tab and inspect the WebSocket connection.

**Comment by mdn522** [comment]
Closing this issue as there is no activity for the last two weeks

**Comment by yateliby** [comment]
Pocket Option was the second one binary options broker where I registered and completed my profile verification. I tried to withdraw my money then and that's when the troubles began. First, they declined my verification. After many attempts, I finally managed to complete it and tried to withdraw the money again. I managed to pass all their levels of verification in the end. But then Pocket Option used their last ace in the hole. They wiped out all of my trading deposit and profits claiming that I was using both to trade and that's prohibited. A few days later, I recovered $79,800 worth of Bitcoin through a triangulation expert at [SilvanaTenreyro@gmx.com](mailto:SilvanaTenreyro@gmx.com). They are best in dealing with online scam and assisting victims in receiving reimbursement, tracking down digital fingerprints, cyber analysis and thorough investigation.

**Comment by PeggyDFluent** [comment]
Like some of you who left a comment, I too was taken in by the Fast Invest adventure. I started investing in January 2021 and I must say that the return on investment was more than conclusive.  Then I wanted to test the withdrawal to make sure everything was OK to continue, and that's when the trouble started. At the time of registration, there were no particular rules, but a few months later, it was impossible to make a full withdrawal of the sums available for investment. We had to proceed in increments of around €47000. Which is what I did every month. I contacted Fast Invest several times at the end of 2021 and the answer was always that they were in difficulty but that the money would be paid out. On 29/12/2023, they closed all withdrawal requests with the reason "return to account". On my account, I can see them in the debts category. I was frustrated until I came across a legal forensic team (mustangggrc@gmail.com) who helped me recover my money. They trace and recover datas, blocked accounts and money. Their powerful recovery tool has empowered me to share their good deed.
